cves/kernel/CVE-2019-20636.yml

```diff
@@ -324,8 +327,10 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: false
+  note: "In the fix for the bug, there was only Dmitry Torokhov in the sign off for
+    it in response to the message received alerting about the vulnerability. It
```

Comment:
This sentence feels a little weird to read. Not sure the best way to rephrase though.